.nxignore

```diff
@@ -0,0 +1,2 @@
+# see comment in the file
+packages/ast-spec/tests/util/parsers/typescript-estree-import.ts
```

Comment:
@JamesHenry - the new version of nx started reporting circular deps, so I had to add this hack back in.